examples/advanced_composition/advanced_composition/map_task.py

```diff
@@ -7,83 +7,85 @@
 # .. tags:: Intermediate
 # ```
 #
-# ```{image} https://img.shields.io/badge/Blog%20Post-Map%20Tasks-blue?style=for-the-badge
-# :alt: Map Task Blog Post
-# :target: https://blog.flyte.org/map-tasks-in-flyte
-# ```
+# Using a map task in Flyte allows for the execution of a pod task or a regular task across a series of inputs within a single workflow node.
+# This capability eliminates the need to create individual nodes for each instance, leading to substantial performance improvements.
```

Comment:
we're okay removing the blogpost link?

Reply:
yep. it was published a long time ago, and i think our docs should suffice.